control/statefbk.py

```diff
@@ -347,6 +347,9 @@ def gram(sys,type):
     #Check for ss system object
     if not isinstance(sys,statesp.StateSpace):
         raise ValueError("System must be StateSpace!")
+    gramtypes = ['c', 'o', 'cf', 'of']
```

Comment:
I don't really remember, I wrote that function a few months ago. If you think there is a better way to do things, then by all means change it.